### PR TITLE
Content box react

### LINKED ISSUE #1037: JSX: content-box

### PR DESCRIPTION
**Pull request by anna-wro**
screen shot 2017-07-10 at 12 46 05
screen shot 2017-07-10 at 12 46 14
screen shot 2017-07-10 at 12 46 21
screen shot 2017-07-10 at 12 46 33
screen shot 2017-07-10 at 12 46 41
screen shot 2017-07-10 at 12 46 47

Closes #1037

### CODE REVIEW

File: src/components/content-box/ContentBoxConstants.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const ALIGNED_SPACE = {
+  TOP: 'top',
+  BOTTOM: 'bottom'
+};
+
+const SPACE_SIZE = {
```
Comment (aju) [inline src/components/content-box/ContentBoxConstants.jsx]: just Size ? how its called in other components?

Comment (aju) [inline src/components/content-box/ContentBoxConstants.jsx]: its Spacing ?

Comment (kdzwinel) [inline src/components/content-box/ContentBoxConstants.jsx]: yeah, lets keep naming consistent 👍

File: src/components/content-box/ContentBoxConstants.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const ALIGNED_SPACE = {
```
Comment (matikrk) [inline src/components/content-box/ContentBoxConstants.jsx]: maybe ALIGNMENT_SPACE? @kdzwinel

File: src/components/content-box/ContentBoxConstants.jsx (hunk 1 above)
Comment (anna-wro) [inline src/components/content-box/ContentBoxConstants.jsx]: I changed it to SIZE

**Pull request merged**